### PR TITLE
Separate application scheme for Cliqz Nightly #265

### LINKED ISSUE #265: Separate application scheme for Cliqz Nightly
Currently Cliqz Nightly uses cliqz:// protocol scheme, which makes it to collide with Cliqz app. If both apps are installed at the same time iOS, will not be able to tell which app should handle the link. 

New scheme may be called `cliqznightly://`

We should parameterize the scheme in same way we parameterize App name, by adding a postfix in the configuration file.

### PR DESCRIPTION
**Pull request by shahenAnt**
Fix #265 

## Implementation details
Separated application scheme for Cliqz Nightly.

## Checklist:

- [ ] My squashed commit messages follows the [7 golden rules](https://chris.beams.io/posts/git-commit/)
- [ ] I updated or created necessary unit tests
- [ ] I've tested my changes in Dark Mode, iPad, iOS <= 12, Landscape and in German

### CODE REVIEW

File: Client/Configuration/AdHoc.xcconfig [hunk 1]
```diff
@@ -2,3 +2,6 @@ BUNDLE_DISPLAY_NAME = $(TARGET_NAME) β
 
 // Bundle Identifier
 APP_BUNDLE_ID = $(PRODUCT_BUNDLE_IDENTIFIER).nightly
+
+// URL Scheme
+CLIQZ_URL_SCHEME = cliqznightly
```
Comment (chrmod) [inline Client/Configuration/AdHoc.xcconfig]: What about Ghostery? The value should probably be in the format of `APP_BUNDLE_ID = $(PRODUCT_BUNDLE_IDENTIFIER).nightly`

File: UserAgent.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -5489,7 +5489,6 @@
 			baseConfigurationReference = 8EF4515D84F7B934A5318AEC /* Pods-Cliqz.adhoc.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon_Adhoc;
-				CLIQZ_URL_SCHEME = cliqz;
```
Comment (chrmod) [inline UserAgent.xcodeproj/project.pbxproj]: It should probably be removed from Ghostery config as well.

File: Client/Configuration/AdHoc.xcconfig (hunk 1 above)
Comment (shahenAnt) [reply Client/Configuration/AdHoc.xcconfig]: Done.

File: UserAgent.xcodeproj/project.pbxproj (hunk 2 above)
Comment (shahenAnt) [reply UserAgent.xcodeproj/project.pbxproj]: Fixed.

**Pull request merged**